bigquery2/bigquery_endpoint.bal

```diff
@@ -0,0 +1,422 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+
+# Bigquery Endpoint object.
+# + bigqueryClient - Bigquery Connector client
+public type Client client object {
+
+    private http:Client bigqueryClient;
+
+    public function __init(BigqueryConfiguration bigqueryConfig) {
+        self.init(bigqueryConfig);
+        self.bigqueryClient = new(BASE_URL, config = bigqueryConfig.clientConfig);
+    }
+
+    # Initialize Bigquery endpoint.
+    #
+    # + bigqueryConfig - Bigquery configuraion
+    public function init(BigqueryConfiguration bigqueryConfig);
+
+    # Lists all projects to which current user have been granted any project role.
+    #
+    # + nextPageToken - Page token, returned by a previous call, to request the next page of results.
+    # + return - ProjectList object on success and error on failure
```

Comment:
Better to add an empty docline between parameters and the return parameter. Can be updated in few other places as well.